examples/dashboard.yml

```diff
@@ -0,0 +1,100 @@
+sources:
+  nyc_taxi:
+    description: Large version of nyc taxi dataset
+    driver: parquet
+    args:
+      urlpath: '{{ CATLOG_DIR }}data/nyc_taxi_wide.parq'
```

Comment:
Bit confused here, ``nyc_taxi_wide.parq`` is still not listed in the datashader datasets.yaml, it is only in the pyviz on.

Reply:
Good point. I think we should probably delete the .csv version; .csv is really not a recommended format for Datashader.